requirements.txt

```diff
@@ -21,5 +21,6 @@ pytest-cov
 nbformat
 papermill
 nteract-scrapbook
-shap
-interpret-core
+gevent>=1.3.6
+interpret-core[required]>=0.1.20, <=0.1.21
+shap>=0.20.0, <=0.34.0
```

Comment:
These three are all taken from:
https://github.com/interpretml/interpret-community/blob/e82fc0904dd999f7aa59c943d956d1dff552108d/python/setup.py#L42

Reply:
yeah, this pin looks better than the interpret-text one, no issues form me